coq/workspace.ml

```diff
@@ -19,8 +19,7 @@ module Setup = struct
   type t =
     { vo_load_path : Loadpath.vo_path list
     ; ml_include_path : string list
-    ; options : (Goptions.option_name * Coqargs.option_command) list
```

Comment:
@artagnon actually I used this type as to provide a finer structure in terms of what command line arguments are passed. For example we could have field records for warnings, for kernel options, etc...

No problem refining that later.

Reply:
`Goptions.option_name * Coqargs.option_command` is not appropriate for non-Vernac options.

Reply:
We can have something more sophisticated like a variant, but as Emilio says no need at the moment.

Reply:
Yes, it is not enough, what I mean is that we should at some point using something a bit finer than string.

`Goptions.option_name` worked fine both for warnings and options tho.